### PR TITLE
Fix typo in RELEASING.md

### PR DESCRIPTION
**Pull request by yukiyuli**
change "only keep termurin variants" to "only keep temurin variants"
Fixes: #3124

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Here are the steps:
 1. Build and Test the OpenJDK for "release" at Adoptium using a build pipeline job as follows:
    - Job: https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk8-pipeline/build (Switch `openjdk8` for your version number)
    - `targetConfigurations`:
-    1. only keep termurin variants
+    1. only keep termurin variatns
```
Comment (karianna) [inline RELEASING.md:130]:
```suggestion
    1. only keep temurin variants
```

**Review by karianna — CHANGES_REQUESTED** [review]
I think you changed the wrong words :-)

**Review by karianna — APPROVED** [review]

**Comment by yukiyuli** [comment]
@karianna Thank you for your correct.

**Comment by yukiyuli** [comment]
@karianna Thank you for your correction.

**Review by gdams — APPROVED** [review]

**Comment by gdams** [comment]
closing as duplicate of https://github.com/adoptium/temurin-build/pull/3128

**Pull request closed without merging**